Makefile

```diff
@@ -210,21 +210,14 @@ job:
 .PHONY: job
 
 kerberos_id ?= dptp
-dry_run ?= true
-force ?= false
-bw_password_path ?= /tmp/bw_password
-kubeconfig_path ?= $(HOME)/.kube/config
+export dry_run ?= true
+export force ?= false
+export kubeconfig_path ?= $(HOME)/.kube/config
 
 # these are useful for dptp-team
-# echo -n "bw_password" > /tmp/bw_password
-# make kerberos_id=<your_kerberos_id> cluster=app.ci ci-secret-bootstrap
+# make cluster=app.ci ci-secret-bootstrap
 ci-secret-bootstrap:
-	$(CONTAINER_ENGINE) pull registry.ci.openshift.org/ci/ci-secret-bootstrap:latest
-	$(CONTAINER_ENGINE) run --rm -v "$(CURDIR)/core-services/ci-secret-bootstrap/_config.yaml:/_config.yaml:z" \
-		-v "$(kubeconfig_path):/_kubeconfig:z" \
-		-v "$(bw_password_path):/_bw_password:z" \
-		registry.ci.openshift.org/ci/ci-secret-bootstrap:latest \
-		--bw-password-path=/_bw_password --bw-user $(kerberos_id)@redhat.com --config=/_config.yaml --kubeconfig=/_kubeconfig --dry-run=$(dry_run) --force=$(force) --cluster=$(cluster) --as=system:admin
+	@./hack/ci-secret-bootstrap.sh
 .PHONY: ci-secret-bootstrap
 
 ci-secret-generator:
```

Comment:
I will revisit this after my changes on the job.

Reply:
Maybe add this target and use it as a pre-step of `ci-secret-generator`? The ugly thing like `- cmd: oc --kubeconfig /tmp/build-farm-credentials/sa.config-updater.$(cluster).config sa create-kubeconfig` need to be taken care of

```
build_farm_credentials_folder ?= /tmp/build-farm-credentials

build_farm_credentials_folder:
	mkdir -p $(build_farm_credentials_folder)
	oc --context app.ci -n ci extract secret/config-updater --to=$(build_farm_credentials_folder) --confirm
.PHONY: build_farm_credentials_folder
```